WPMedia/fusion-news-theme-blocks
# ASUB-8303/ Checkout card detail

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

- Create CheckoutCardDetail component

## Jira Ticket

- https://arcpublishing.atlassian.net/browse/ASUB-8303

## Acceptance Criteria

AC:
- Create the CheckoutCardDetail with Title, Summary, Link & Children, following the list provided above (a-e)
- Create an instance of CheckoutCardDetail for 1. Account following the logic defined below. 
- The children will contain the LogIn components that already exist. (check: LogIn, login-links, social-sign-on, …) 
## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout {branch name}`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/subscriptions-block`
3. If a user is logged in, it should open the billing address card
4. If a user is not logged in, it should redirect the user to the log in page and redirect back to checkout once user is logged in.

## Effect Of Changes

### Before

_Example: When I clicked the search button, the button turned red._

[include screenshot or gif or link to video, storybook would be awesome]

### After

_Example: When I clicked the search button, the button turned green._

[include screenshot or gif or link to video, storybook would be awesome]

## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
